src/datastore/common.ts

```diff
@@ -348,20 +343,42 @@ export interface AddressNftEventIdentifier {
   tx_id: Buffer;
 }
 
+export interface AddressTxUpdateEventInfo {
+  amount: string;
+  sender?: string;
+  recipient?: string;
+}
+
+export interface AddressTxUpdateInfoTxDetails {
+  block_height: number;
+  stx_events: AddressTxUpdateEventInfo[];
+}
+
+export interface AddressTxUpdateInfo {
+  address: string;
+  txs: Record<string, AddressTxUpdateInfoTxDetails>;
+}
+
+export interface TokenMetadataUpdateInfo {
+  queueId: number;
+  txId: string;
+  contractId: string;
+}
+
 export type DataStoreEventEmitter = StrictEventEmitter<
   EventEmitter,
   {
-    txUpdate: (info: DbTx | DbMempoolTx) => void;
+    txUpdate: (txId: string) => void;
     blockUpdate: (
-      block: DbBlock,
+      blockHash: string,
       txIds: string[],
```

Comment:
@zone117x do you think I could also remove this array from this message and get the `tx_id`s later? Not sure if reorgs affect these results.

I'm afraid that if a block with hundreds of txs comes along, this array will be enough to break the 8000 byte limit for postgres NOTIFY. What do you think?

Reply:
Hmm yeah, seems like this could exceed the 8000 byte limit and should be removed.

> Not sure if reorgs affect these results

Shouldn't have an issue here. A given block hash should always have the same set of txs.

Reply:
Great. I'll make the change.